### PR TITLE
Umberto integration - cleanup tailing HTML

### PR DESCRIPTION
**Pull request by mlewand**
In PR #121 quite a few files contained random closing HTML markers at the end. This PR corrects that.

### CODE REVIEW

File: docs/guide/dev/features/accessibility_checker/README.md [hunk 1]
```diff
@@ -295,4 +295,3 @@ Refer to the following resources for more information about accessibility in CKE
 * The {@link guide/dev/features/shortcuts/README Keyboard Shortcuts} article lists all keyboard shortcuts supported in CKEditor.
 * The {@link guide/dev/features/tabindex/README Page Navigation Using the "Tab" Key} article discusses how CKEditor participates in the page <kbd>Tab</kbd> order.
 * The {@link guide/dev/section508/README CKEditor Section 508 Compliance} article describes CKEditor 4 compatibility with [Section 508 Amendment to the Rehabilitation Act of 1973](http://www.state.gov/m/irm/impact/c32157.htm).
-* The {@link guide/dev/wcag/README CKEditor WCAG 2.0 Compliance} article describes CKEditor 4 compatibility with the [Web Content Accessibility Guidelines (WCAG) 2.0](http://www.w3.org/TR/WCAG20/) standard.</img>
```
Comment (m-turek) [inline docs/guide/dev/features/accessibility_checker/README.md]: The link is removed intentionally?

File: docs/guide/dev/features/pastefromword/README.md [hunk 2]
```diff
@@ -119,4 +119,3 @@ Refer to the following resources for more information about pasting content:
 * The {@link guide/dev/deep_dive/clipboard/README Clipboard Integration} article explains how Clipboard API is implemented in CKEditor.
 * The {@link guide/dev/configuration/file_browse_upload/file_upload/README Uploading Dropped or Pasted Files} article describes drag&drop in CKEditor.
 * The {@link guide/dev/configuration/acf/README Content Filtering (ACF)} is an introduction to CKEditor's unique content filtering system.
-* The {@link guide/dev/features/styles/README Applying Styles to Editor Content} article discusses creating more semantically correct text styles.</img>
```
Comment (m-turek) [inline docs/guide/dev/features/pastefromword/README.md]: Again, whole link removed

File: docs/guide/dev/upgrade/upgrade_fckeditor_2/README.md [hunk 3]
```diff
@@ -21,4 +21,3 @@ However, since CKEditor was a complete rewrite of FCKeditor, with lots of change
 The best option is to [download the latest CKEditor version](https://ckeditor.com/ckeditor-4/download/) and install it, starting anew, as described in our {@link guide/dev/installation/README Quick Start Guide}.
 
 <info-box hint=""> If you would like to try to mimic your FCKeditor configuration in CKEditor, see the <a href="http://docs.cksource.com/CKEditor_3.x/Developers_Guide/FCKeditor_CKEditor_Configuration_Mapping">matrix of the mappings</a> between these two.
-</info-box>
```
Comment (m-turek) [inline docs/guide/dev/upgrade/upgrade_fckeditor_2/README.md]: This one is needed

File: docs/guide/dev/wcag/README.md [hunk 4]
```diff
@@ -675,4 +675,3 @@ Refer to the following resources for more information about accessibility in CKE
 * The {@link guide/dev/features/accessibility_checker/README Managing Content Accessibility with Accessibility Checker} article describes an innovative Accessibility Checker tool that lets you inspect the accessibility level of content created in CKEditor and immediately solve any issues that are found.
 * The {@link guide/dev/features/shortcuts/README Keyboard Shortcuts} article lists all keyboard shortcuts supported in CKEditor.
 * The {@link guide/dev/features/tabindex/README Page Navigation Using the "Tab" Key} article discusses how CKEditor participates in the page <kbd>Tab</kbd> order.
-* The {@link guide/dev/section508/README CKEditor Section 508 Compliance} article describes CKEditor 4 compatibility with [Section 508 Amendment to the Rehabilitation Act of 1973](http://www.state.gov/m/irm/impact/c32157.htm).</br>
```
Comment (m-turek) [inline docs/guide/dev/wcag/README.md]: Removed link

File: docs/guide/index.md [hunk 5]
```diff
@@ -42,4 +42,3 @@ meta-description: Learn how to install, integrate, configure and develop CKEdito
 <dt>{@link guide/skin_sdk/intro/README Introduction}</dt><dd>Creating custom CKEditor skins.</dd>
 <dt>{@link guide/skin_sdk/icons/README Advanced Concepts}</dt><dd>In-depth details about concepts that constitute a skin.</dd>
 <dt>{@link guide/skin_sdk/setup/README Creating a Custom Skin}</dt><dd>The workflow of creating a custom skin.</dd>
-</dl>
```
Comment (m-turek) [inline docs/guide/index.md]: It should probably stay

File: docs/guide/dev/features/pastefromword/README.md (hunk 2 above)
Comment (mlewand) [reply docs/guide/dev/features/pastefromword/README.md]: Got picked up due to trailing `</img>`, fixing.

File: docs/guide/dev/features/accessibility_checker/README.md (hunk 1 above)
Comment (mlewand) [reply docs/guide/dev/features/accessibility_checker/README.md]: Got picked up due to trailing `</img>`, fixing.

**Pull request merged**